src/main/groovy/com/codebrig/arthur/observe/structure/StructureLiteral.groovy

```diff
@@ -40,6 +60,22 @@ abstract class StructureLiteral {
         return getNodeLiteralAttribute(node) != null
     }
 
+    boolean isNegative(SourceNode node) {
```

Comment:
If `isNegative()` is only used by one function it should be inline. If it's used by more then maybe a clearer name should be used (`isNodeLiteralNegative()`).

Reply:
Resolved with commit https://github.com/CodeBrig/Arthur/pull/79/commits/51f0884fa52bfb80b0d83652785a5198d3fff374